client/state/happychat/constants.js

```diff
@@ -17,6 +17,12 @@ export const HAPPYCHAT_CONNECTION_STATUS_UNAUTHORIZED = 'unauthorized';
 // Max number of messages to save between refreshes
 export const HAPPYCHAT_MAX_STORED_MESSAGES = 30;
 
+// Skills
+export const HAPPYCHAT_SKILLS = {
```

Comment:
I'm used to using the `Go to definition` feature instead of greping. I like objects because they make imports and adding new constants easier but I'll replace them with more verbose constants if that helps discoverability.